extensions-contrib/opentelemetry-emitter/pom.xml

```diff
@@ -161,6 +165,7 @@
         <configuration>
           <ignoredUnusedDeclaredDependencies>
             <!-- Transitive dependencies from opentelemetry but explicitly added to be shadowed -->
+            <ignoredUnusedDeclaredDependency>io.opentelemetry:opentelemetry-exporter-otlp</ignoredUnusedDeclaredDependency>
```

Comment:
instead of ignoring it, it might  be enough to make opentelemetry-exporter-otlp a `runtime` scope dependency to make it clear it's not used at compile time but required at runtime.

Reply:
yeah, sounds good
I tried to push a commit to this PR, but I don't have permissions 
`Permission to confluentinc/druid.git denied to IvanVan.`

Also I don't know how to update PR upstream, because there is the same issue there and I want to add info about `tracestate`
I can create another PR to upstream from my own fork
What do you think of it? @xvrl